File: TOPH ONLINE JUDGE PROBLEM SOLUTION IN JAVA/Easy Problems/Running Average.java
import java.util.Scanner;
public class Solution{
public static void main(String[] args){
	Scanner input=new Scanner(System.in);
	int i, T,num;
	double sum,avg;
	T=input.nextInt();
	sum=0;
	for(i=1;i<=T;i++){
		num=input.nextInt();
		sum=sum+num;
		avg=sum/i;
		System.out.println(avg);
	}
}
}

File: TOPH ONLINE JUDGE PROBLEM SOLUTION IN c/Easy Problems/Running Average.c
#include <stdio.h>

int main() {
	int T,i,num;
	double avg,sum;
	
		scanf("%d",&T);
sum=0;
	for(i=1;i<=T;i++){
		scanf("%d",&num);
		sum=sum+num;
		avg=sum/i;
	   printf("%lf\n",avg);
	}
	return 0;
}
